### DISCUSSION TITLE
Oracle - Use bind variables with slick.lifted

### DISCUSSION THREAD

**Discussion by NicoGrdSG** [slick/slick#3000] [Questions]
Hello,

I have a question about Oracle db and hard parses.

I have a Scala service that's connected to an Oracle db that I'm querying using the slick.lifted interface. Slick version is 3.4.1, Oracle JDBC is 23.4.0.24.05.
Basically my queries look like this:
```
val query = for {
      myFilter <- myTableQuery.filter(x => x.value === myValue)
  } yield (myFilter)

  db.run(query.result)
```

My problem is: all these generated queries are sent to Oracle via hard parses . To optimize my calls, I would like to know if it is possible to configure Slick (or update my code) to send queries with bind variables, instead of these hard parses, without having to write plan SQL queries.

Thank you very much,
Nicolas

**Comment by nafg** [comment]
What do you mean exactly by "hard parses"?
